scripts/linux/install.sh

```diff
@@ -10,6 +10,12 @@ then
     exit -1
 fi
 
+if [ -n "$DSN_ROOT" -a -f "$DSN_ROOT/lib/libdsn.core.so" ]
```

Comment:
only the first time, to set the env.

On Fri, Jul 8, 2016 at 9:48 AM, Zhenyu Guo <notifications@github.com> wrote:

> In scripts/linux/install.sh
> <https://github.com/imzhenyu/rDSN/pull/451#discussion_r70015958>:
>
> > @@ -10,6 +10,12 @@ then
> >      exit -1
> >  fi
> >
> > +if [ -n "$DSN_ROOT" -a -f "$DSN_ROOT/lib/libdsn.core.so" ]
>
> Confused. make install does not work anymore?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/imzhenyu/rDSN/pull/451/files/b1675a27aba609664e3ff465bfd7617d52f5be18#r70015958>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAgQPQXmPFQiSNMx8BZc4C4QTlJfKLrGks5qTaxTgaJpZM4JG8QB>
> .
>